.gitignore

```diff
@@ -53,4 +53,6 @@ Carthage/Build
 ### warning!
 ### do not ignore Carthage/Checkouts. see https://github.com/Carthage/Carthage/issues/1153
 ###Carthage/Checkouts  
-###
\ No newline at end of file
+###
+
+Pods/
```

Comment:
1. Let's add a newline here.
1. When I check out this branch, there's an untracked "Carthage" folder. I think we should either ignore it, or check in its deletion, since we're not using Carthage any more _locally_.

Reply:
I added a Gemfile, deleted everything Carthage. I still haven't fixed the unit-test target, struggling there.

Reply:
Happy to pair on it on Monday or thereabouts